sdk/spring/spring-cloud-azure-integration-tests/src/test/java/com/azure/spring/cloud/integration/tests/eventhubs/EventHubsIT.java

```diff
@@ -74,6 +74,8 @@ EventHubsErrorHandler errorHandler() {
     @Test
     public void testEventHubOperation() throws InterruptedException {
         LOGGER.info("EventHubsIT begin.");
+        // Wait for eventhub initialization to complete
+        Thread.sleep(20000);
```

Comment:
Can we add re-try logic for this kind of tests?

Reply:
I assume it's required to retry the application context runner, not a small logic in the test method, currently, seems there's no suitable solution.

Reply:
I assume it's required to retry the whole execution of the application test context runner, not a small logic in the test method, currently, seems there's no suitable solution. Will have a try in another PR.